src/app/clarity/descriptions/Description.m.scss

```diff
@@ -84,3 +84,100 @@
 .breakSpaces {
   white-space: break-spaces;
 }
+
+//--- images
+@mixin img_before {
+  background-repeat: no-repeat;
+  background-position: center;
+  display: inline-block;
+  content: '';
+  margin: 0 1px;
+  height: 10px;
+}
+.arc::before,
+.solar::before,
+.void::before,
+.stasis::before,
+.strand::before,
+.barrier::before,
+.overload::before,
+.unstoppable::before {
+  background-size: 10px 10px;
+  width: 10px;
+}
+.primary::before,
+.special::before,
+.heavy::before {
+  background-size: 18px 20px;
+  width: 13px;
+}
+.warlock::before {
+  width: 22px;
+}
+.hunter::before {
+  width: 10px;
+}
+.titan::before {
+  width: 11px;
+}
+
+//--- elements
+.arc::before {
+  background-image: url(https://www.bungie.net/common/destiny2_content/icons/DestinyDamageTypeDefinition_092d066688b879c807c3b460afdd61e6.png);
```

Comment:
Ahh yes completely forgotten about this
Because of how it works it's not img tags can't replace with component with out checking if class name is arch for example
I was thinking about using css variables for link's
I will probably take a look later today